dataclients/kubernetes/definitions/testdata/validation/route-with-multiple-filters-one-item.log

```diff
@@ -1 +1 @@
-single filter expected
+single filter expected at "foo(42) -> bar(24)"
```

Comment:
I don't quite understand, what Skipper user should do when he/she wants to have 2+ filters for single route? It seems to me it's valid use case.

Reply:
This is 2 filters in one yaml item
```yaml
filters:
  - foo(42) -> bar(24)
```

We don't want this, and also this is a test case from another [PR](https://github.com/zalando/skipper/pull/2691) that was already merged.

We want people to use

```yaml
filters:
  - foo(42)
  - bar(24)
```